### PR TITLE
Package Factoring - New ColoredConsole Sink Project

### PR DESCRIPTION
**Pull request by merbla**
Related to #593.  Jumping off the back of #630.

Might need some fine tuning post #630 getting merged.

### CODE REVIEW

File: src/Serilog.Sinks.ColoredConsole/project.json [hunk 1]
```diff
@@ -0,0 +1,24 @@
+﻿{
+  "version": "2.0.0-beta-*",
+  "description": "The colored console sink for Serilog",
+  "authors": [ "Serilog Contributors" ],
+  "tags": [ "serilog", "console", "coloredconsole" ],
+  "projectUrl": "http://serilog.net",
+  "licenseUrl": "http://www.apache.org/licenses/LICENSE-2.0",
+  "iconUrl": "http://serilog.net/images/serilog-nuget.png",
```
Comment (nblumhardt) [inline src/Serilog.Sinks.ColoredConsole/project.json]: Icon should be: http://serilog.net/images/serilog-sink-nuget.png

**Comment by merbla** [comment]
now #630 is done. Will update ready for merge.

**Comment by merbla** [comment]
@khellang @nblumhardt moar squirrels?

**Comment by khellang** [comment]
Yep. LGTM :shipit:

**Comment by khellang** [comment]
A squash might be warranted, if we care about that stuff :stuck_out_tongue:

**Comment by merbla** [comment]
@khellang on the PR or on the way into dev?

**Comment by khellang** [comment]
Either :wink: It's probably easier if you do it, so @nblumhardt can just push the GH button :smile:

**Comment by merbla** [comment]
ok will try to do before I jump on my :airplane: , otherwise when I am back :house_with_garden: in Oz.

**Comment by merbla** [comment]
Cancelling this as a larger sink PR is incoming.

**Pull request closed without merging**